plugins/api-docs/api-report.md

```diff
@@ -28,7 +28,7 @@ export const ApiDefinitionCard: () => JSX.Element;
 export type ApiDefinitionWidget = {
   type: string;
   title: string;
-  component: (definition: string) => React_2.ReactElement;
+  component: (entity: ApiEntity) => React_2.ReactElement;
```

Comment:
Wonder if we should deprecate the `definition` param and then grab the entity with `useEntity` for these instead? That way we avoid the breaking change for now and we eventually reduce the API surface.